infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -179,7 +187,7 @@ param parVpnGatewayConfig object = {
 param parExpressRouteGatewayConfig object = {
   name: '${parCompanyPrefix}-ExpressRoute-Gateway'
   gatewayType: 'ExpressRoute'
-  sku: 'ErGw1AZ'
+  sku: 'ErGw2AZ'
```

Comment:
Why the change?

Reply:
I believe this was for further testing...reverted